cli/translate_outputs.py

```diff
@@ -58,32 +56,48 @@ def translate_parser_outputs(
     _target_languages = set(TARGET_LANGUAGES)
 
     for path in tqdm(task_output_paths):
-        _LOGGER.debug(f"Translator processing - {path}.")
+        _LOGGER.debug(
+            "Translator processing document.",
+            extra={"props": {"path": f"{path}"}},
+        )
 
         try:
             parser_output = ParserOutput.parse_raw(path.read_text())
             _LOGGER.debug(
-                f"Successfully parsed {path} from output dir during translation processing."
+                "Successfully parsed document from output dir during translation processing.",
+                extra={"props": {"path": f"{path}"}},
             )
-        except FileNotFoundError:
+        except FileNotFoundError as e:
             _LOGGER.error(
-                f"Could not find {path} in output dir during translation processing."
+                "Could not find document in output dir during translation processing.",
+                extra={
+                    "props": {
+                        "path": f"{path}",
+                        "error": str(e),
+                    }
+                },
             )
             continue
 
         if should_be_translated(parser_output):
-            _LOGGER.info(f"Document should be translated: {path}")
+            _LOGGER.info(
+                "Document should be translated if target languages.",
+                extra={"props": {"path": f"{path}"}},
+            )
 
             target_languages = identify_translation_languages(
                 parser_output, _target_languages
             )
-            _LOGGER.debug(f"Target languages: {target_languages} for {path}")
+            _LOGGER.debug(
+                "Target languages identified.",
+                extra={"props": {"target_languages": target_languages}},
+            )
 
             _translate_to_target_languages(
                 path, parser_output, target_languages, redo=redo
             )
 
-    _LOGGER.info("Finished translation stage for ALL input tasks.")
+    _LOGGER.info("Finished translation stage for all input tasks.")
```

Comment:
Isn't there anymore context we can give to this message - i.e. what context are used to reference the input task?